packages/core/src/components/table/table-header-cell/table-header-cell.tsx

```diff
@@ -201,6 +208,7 @@ export class TdsTableHeaderCell {
         <button class="tds-table__header-button" onClick={() => this.sortButtonClick()}>
           <span class="tds-table__header-button-text" style={{ textAlign: this.textAlignState }}>
             {this.cellValue}
+            <slot />
```

Comment:
Great stuff. Should we update the CSS to style this element also? I think not, the slots to me are for advanced users and we should not dabble in their stuff.

Reply:
I agree, I think styles should not be part of it. Do you see some of styles that should be removed?